File: Zhukov/src/Hangman/Common/LettersPoolRender.java
package Hangman.Common;

import Hangman.LettersPool;

import java.util.List;

public class LettersPoolRender {
    public void show(LettersPool lettersPool) {
        List<Character> list = lettersPool.toList();
        System.out.print("Использованные буквы: ");
        for (Character c : list) {
            System.out.print(c + " ");
        }
        System.out.println();
    }
}

File: Common/LettersPoolRender.cs
namespace Hangman.Common
{
    internal class LettersPoolRender
    {
        public void show(LettersPool pool)
        {
            List<Char> list = pool.toList();
            Console.Write("Использованные буквы: ");
            for (int i = 0; i < list.Count; i++) { Console.Write(list[i] + " "); }
            Console.WriteLine();
        }
    }
}
